### ISSUE TITLE
Recitation 09-extra pdf not on luminus

### ISSUE THREAD

**Issue by neverdivideby0** [nus-cs2030/2122-s2#595]
Hi not sure if its just me but as i just started watching Recitation 8 but Prof Henry is using Recitation 09-extra PDF but i cant find it on luminus

**Comment by jamietan123** [comment]
I don't think theres a recitation 9?

**Comment by neverdivideby0** [comment]
Yeah there isnt a recitation 9 video but there seems to be a recitation 9-extra file if you watch recitation 8 I THINK...
